src/js/jsx/shared/TextInput.jsx

```diff
@@ -138,6 +138,15 @@ define(function (require, exports, module) {
                 });
             }
         },
+        
+        /**
+         * Return the text input's current value
+         * 
+         * @return {string}
+         */
+        getValue: function () {
```

Comment:
Added getter function `getValue` so that the parent component does not need to listen to its change event or keep a copy of the current value.